### PR TITLE
move location of settings.ini to outside of the repo to avoid accidental committing.

### PR DESCRIPTION
**Pull request by akash1810**
move location of settings.ini to outside of the repo to avoid accidental committing.

### CODE REVIEW

**Comment by theefer** [comment]
:+1:

**Comment by jamesgorrie** [comment]
Eventually we should remove it from the `.gitignore` too.

File: scripts/dot-properties/main.py [hunk 1]
```diff
@@ -21,11 +22,9 @@ def main():
     if not cf_stack:
         raise Exception('No stack found in {}'.format(CONFIG_FILE))
 
-    output_dir = _default_option(config, 'output', 'directory', 'output')
-
     properties = config._sections['properties']
 
-    generate_files(aws_profile_name, aws_region, cf_stack, output_dir, properties)
+    generate_files(aws_profile_name, aws_region, cf_stack, OUTPUT_DIR, properties)
```
Comment (tsop14) [inline scripts/dot-properties/main.py]: Is there a convention for when to use lower case and when to use upper case?

**Comment by akash1810** [comment]
> Eventually we should remove it from the `.gitignore` too.

Yep - after everyone's deleted the file first!

File: scripts/dot-properties/main.py (hunk 1 above)
Comment (akash1810) [inline scripts/dot-properties/main.py]: The python convention is upper-case for constants and lower for variables.

**Pull request merged**